### ISSUE TITLE
Users module failing at retrieving metadata?

### ISSUE THREAD

**Issue by chiefGui** [magiclabs/magic-admin-js#68]
### ✅ Prerequisites

- [x] Did you perform a cursory search of open issues? Is this bug already reported elsewhere?
- [x] Are you running the latest SDK version?
- [x] Are you reporting to the correct repository (`@magic-sdk/admin`)?

### 🐛 Description

[Description of the bug.]

### 🧩 Steps to Reproduce

```ts
import { Magic } from '@magic-sdk/admin' // v1.3.0

const mAdmin = new Magic(SECRET_KEY)

export default async function(req, res) {
  const didToken = mAdmin.utils.parseAuthorizationHeader(req.headers.authorization)
  
  try {
    const issuer = await mAdmin.token.getIssuer(didToken) // works!
    const metadata = await mAdmin.users.getMetadataByIssuer(issuer) // nope.
  } catch (e) {
    console.error(e) // {"code":"SERVICE_ERROR","data":[{"data":{},"error_code":"MISSING_REQUIRED_FIELDS","message":"Missing required fields: ['issuer'].","status":"failed"}]}
  }
}
```

### 🤔 Expected behavior

Wanted to retrieve user's metadata.

### 😮 Actual behavior

Am consistently receiving 400 and no metadata.

### 💻 Code Sample

```ts
import { Magic } from '@magic-sdk/admin' // v1.3.0

const mAdmin = new Magic(SECRET_KEY)

export default async function(req, res) {
  const didToken = mAdmin.utils.parseAuthorizationHeader(req.headers.authorization)
  
  try {
    const issuer = await mAdmin.token.getIssuer(didToken) // works!
    const metadata = await mAdmin.users.getMetadataByIssuer(issuer) // nope.
  } catch (e) {
    console.error(e) // {"code":"SERVICE_ERROR","data":[{"data":{},"error_code":"MISSING_REQUIRED_FIELDS","message":"Missing required fields: ['issuer'].","status":"failed"}]}
  }
}
```

### 🌎 Environment

| Software           | Version(s) |
| ------------------ | ---------- |
| `@magic-sdk/admin` | v1.3.0
| Node               | v15.14.0
| `yarn`             | v3.1.0
| Operating System   | macOS

**Comment by chiefGui** [comment]
Weird enough, I tried two things:

```ts
await fetch('https://api.magic.link/v1/admin/auth/user/get?issuer=did%3Aethr%SOME_ID_HERE', { method: 'GET', headers: { 'X-Magic-Secret-Key': SECRET_KEY }})
```
Works ✅ 

```ts
const mAdmin = new Magic(SECRET_KEY, {
  endpoint: 'https://api.magic.link',
})
```
Doesn't work 😢 

### Notes:

1. `.validate` is working properly.
2. The exact same error occurred with `getMetadataByToken` as well.

**Comment by chiefGui** [comment]
Okay, I just dove deep and yes, it seems it's the SDK:

https://github.com/magiclabs/magic-admin-js/blob/731a8cb440533b459eefc70d29d9f027e1db29fa/src/utils/rest.ts#L34

**Comment by smithki** [comment]
Thanks for you detailed report and investigation (and further initiative on a fix/PR). This is now released in `@magic-sdk/admin@1.3.1`
